packages/manager/apps/pci-quota/src/pages/regions/Regions.page.tsx

```diff
@@ -0,0 +1,169 @@
+import { OdsButton, OdsMessage, OdsText } from '@ovhcloud/ods-components/react';
+import { useEffect, useState } from 'react';
+import { Translation, useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import {
+  Notifications,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
+import {
+  isDiscoveryProject,
+  PciAnnouncementBanner,
+  PciDiscoveryBanner,
+  useProject,
+} from '@ovh-ux/manager-pci-common';
+import { useMedia } from 'react-use';
+import { AvailablePart } from '@/pages/regions/Available.part';
+import { ToAddPart } from '@/pages/regions/ToAdd.part';
+import { useGetAvailableRegions, useLocations } from '@/api/hooks/useRegions';
+import { addRegion } from '@/api/data/region';
+import queryClient from '@/queryClient';
+
+export type TPlainLocation = {
+  name: string;
+  continent: string;
+  regions: string[];
+};
+
+type TState = {
+  isAddingRegion: boolean;
+};
+
+export default function RegionsPage(): JSX.Element {
+  const { t } = useTranslation('regions');
+
+  const [state, setState] = useState<TState>({ isAddingRegion: false });
```

Comment:
Please use unit state 
```suggestion
  const [isAddingRegion, setIsAddingRegion] = useState<boolean>(false);
```